### PR TITLE
Fixing visual optimization bugs2

### PR DESCRIPTION
**Pull request by Braiam23**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [X] 🐛 Bug Fix
- [ ] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [ ] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

After the optimization, it seems that several visual bugs were caused, luckily most of them can be fixed.

## 🎯 Key Objectives

Fix the damn bugs, this mod looks like Call of Duty for each fix or optimization the mod breaks more

## 🚦 Testing 

Yep everything is tested before uploading this to the branch
Note: It is not necessary to make a Changelog because this should not happen. It is supposed that after the optimization the mod should remain intact, not cause more bugs._.

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [X] 🙅 no documentation needed

If in case, it causes a problem, let me know so I can close the pull request
If this happens, just open a pull request with the projectile models since that does not touch anything in the general code

### CODE REVIEW

**Comment by Braiam23** [comment]
@Desoroxxx One question, are these new lines of code really related to the blockbench one?

image

File: src/main/java/com/paneedah/mwc/items/guns/F2000Factory.java [hunk 1]
```diff
@@ -427,9 +414,6 @@ else if(model instanceof AcogReticle) {
                 .setupModernMagazineAnimations("f2000", 
                 		Magazines.M4A1Mag, 
                 		Magazines.M38Mag, 
-                		Magazines.Stanag100,
-                		Magazines.Stanag50,
-                		Magazines.Stanag60,
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/F2000Factory.java:432]: Why are they getting removed?

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/paneedah/mwc/items/guns/F2000Factory.java (hunk 1 above)
Comment (Braiam23) [reply src/main/java/com/paneedah/mwc/items/guns/F2000Factory.java:432]: For reasons of realism in real life, the F2000 does not support Drum magazine.

Comment (Braiam23) [reply src/main/java/com/paneedah/mwc/items/guns/F2000Factory.java:432]: The maximum compatible would be a 40 or 45 bullet charger but it does not exist

**Comment by Desoroxxx** [comment]
> @Desoroxxx One question, are these new lines of code really related to the blockbench one?
> 
> image

Huh can you use GitHub links or elaborate just randomly showing me a picture with 3.5 lines of codes won't give me much info

**Pull request merged**